### PR TITLE
Preparation for jpm

### PR DESCRIPTION
**Pull request by bmaschio**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.idea/*
```
Comment (klag) [inline .gitignore]: Please remove this file from the commit

File: .idea/compiler.xml [hunk 2]
```diff
@@ -1,13 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (klag) [inline .idea/compiler.xml:1]: Please remove this file from the commit

File: pom.xml [hunk 3]
```diff
@@ -71,6 +59,67 @@
                     <target>8</target>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```
Comment (klag) [inline pom.xml:63]: what about this, why you need it?

File: rabbitmq-javaservice.iml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (klag) [inline rabbitmq-javaservice.iml]: is it necessary this file in repo?

File: src/main/java/org/jolie/rabbitmq/RabbitMQJavaService.java [hunk 5]
```diff
@@ -265,35 +265,37 @@ private void getMessage(){
                 GetResponse response = null;
```
Comment (klag) [inline src/main/java/org/jolie/rabbitmq/RabbitMQJavaService.java:266]: it seems to me that this file has not been changed, right?

File: test/client.ol [hunk 6]
```diff
@@ -31,4 +33,8 @@ main {
         .message.pippo.pluto = args[ 0 ]
     }
     writeOnExchange@RabbitMQ( message )
-}
\ No newline at end of file
+}
+
+
+}
+
```
Comment (klag) [inline test/client.ol:40]: the following four jars, are they downloaded with maven? if yes, you can remove it from the commit

**Review by klag — COMMENTED** [review]
Please check the inline comments

File: pom.xml (hunk 3 above)
Comment (bmaschio) [reply pom.xml:63]: to prepare the project to be published on jpm

File: .gitignore (hunk 1 above)
Comment (bmaschio) [reply .gitignore]: what?

File: test/client.ol (hunk 6 above)
Comment (bmaschio) [reply test/client.ol:40]: @klag shell I remove all the jar then?

File: .gitignore (hunk 1 above)
Comment (bmaschio) [reply .gitignore]: @klag they been all removed by 697618a

File: rabbitmq-javaservice.iml (hunk 4 above)
Comment (bmaschio) [reply rabbitmq-javaservice.iml]: @klag  no

File: .gitignore (hunk 1 above)
Comment (klag) [reply .gitignore]: here there is the file .gitignore

File: test/client.ol (hunk 6 above)
Comment (klag) [reply test/client.ol:40]: mmmm, the jar should be released within the folder lib. Does jpm do it? If the answer is yes, we can remove them from the lib folder of the project.

Comment (klag) [reply test/client.ol:40]: The test should be modified in order to use jpm for downloading the right libraries

**Review by klag — APPROVED** [review]

**Pull request merged**